packages/access-api/src/config.js

```diff
@@ -1,5 +1,7 @@
 import { Signer } from '@ucanto/principal/ed25519'
 import * as DID from '@ipld/dag-ucan/did'
+// eslint-disable-next-line no-unused-vars
+import * as dagUcan from '@ipld/dag-ucan'
```

Comment:
Thanks, I updated to that, and will prefer importing from ucanto libs in the future instead of dag-ucan.
```
import { UCAN } from '@ucanto/interface';
import { DID } from '@ucanto/core';
```

Reply:
https://github.com/web3-storage/w3protocol/pull/334/commits/fcc80ea334dd9aaacb94c454caa7351543a5dc6e
https://github.com/web3-storage/w3protocol/pull/334/commits/9559d0e13445a96be3009a641f35658eb3fed0e1